data/countries/ZA.yaml

```diff
@@ -90,3 +90,7 @@ holidays:
       substitutes 12-26 if sunday then next monday:
         name:
           en: Public Holiday
+      2023-12-15:
```

Comment:
Please put the date in quotes. Yaml parses this as a datetime instead of a required string.
"2023-12-15": will do.